Makefile

```
@@ -51,6 +51,25 @@ conformance-failing-list: poetry
 		sed -i '1{/^[[:space:]]*$$/d;}' tests/failing.llvm ;\
 	fi
 
+download-json-fixtures:
+	rm -rf tests/execution-spec-tests/fixtures
+	cd tests/execution-spec-tests ;\
+	bash get_execution_spec_tests.sh
```

```suggestion
	cd tests/execution-spec-tests && bash get_execution_spec_tests.sh
```

Using `&&` ensures that if the first command fails (if the directory isn't there), then the whole line will fail.